pages/common/aws-secretsmanager.md

```diff
@@ -7,13 +7,17 @@
 
 `aws secretsmanager list-secrets`
 
+- List all secrets but only show the secret names and ARNs (easy to view):
+
+`aws secretsmanager list-secrets --query 'SecretList[*].{Name: Name, ARN: ARN}'`
+
 - Create a secret:
 
-`aws secretsmanager create-secret --name {{name}} --description "{{secret_description}}" --secret-string {{secret}}`
+`aws secretsmanager create-secret --name {{name}} --description "{{secret_description}}" --secret-string '{{secret}}'`
 
-- Delete a secret:
+- Delete a secret (append `--force-delete-without-recovery` to delete immediately without any recovery period):
 
-`aws secretsmanager delete-secret --secret-id {{name_or_arn}}`
+`aws secretsmanager delete-secret --secret-id {{name|arn}}`
```

Comment:
```suggestion
`aws secretsmanager delete-secret --secret-id {{name|arn}} {{--force-delete-without-recovery}}`
```

Reply:
I don't think we need this change as the example description already says, inside brackets, to as this option if the user doesn't want recovery.

Reply:
As @fazlearefin already described, I don't think this change is needed. Can be closed tbh

Reply:
> I don't think we need this change as the example description already says, inside brackets, to add this option if the user doesn't want recovery. 

I know but I seen similar example in other page, and the optional flag was written in the placeholder syntax too